library/coverage/result_spec.rb

```diff
@@ -0,0 +1,34 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe 'Coverage.result' do
+  before :all do
+    @config_file = File.expand_path('../fixtures/start_coverage.rb', __FILE__)
+  end
+
+  after :each do
+    $LOADED_FEATURES.delete(@config_file)
+  end
+
+  it 'gives the covered files as a hash with arrays' do
+    require 'coverage'
```

Comment:
`require 'coverage'` should be done only once, on top for `library` specs (like in other files).